### ISSUE TITLE
Sport-Études

### ISSUE THREAD

**Issue by Crocmagnon** [BdEINSALyon/marsu#6]
How do we deal with Sport-Études section and their 3 years of PC ?

**Comment by PhilippeGeek** [comment]
There is no 3rd year, there is 0, 1 and 2 for Sport-Études section

**Comment by Crocmagnon** [comment]
Yes I know, we need to add a "0A" (and "5A" by the way) to fixtures

**Comment by PhilippeGeek** [comment]
Yes it can be more suitable

**Comment by Crocmagnon** [comment]
Still, the default should point to 1A
